docs/app/pages/components/components-sections/input-controls/number-picker/number-picker.component.html

```diff
@@ -41,6 +41,10 @@
         	Defines the maximum value the number picker can set.
     </tr>
     <tr uxd-api-property name="valid" type="boolean">
+        <p><b>Note:</b> This component has been deprecated. We recommend using <a
```

Comment:
This should be "property" rather than "component." Also, could you use `href` instead of `routerLink` for the external URL? Thanks.

Reply:
Updated